fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/query/expressions/QueryExpressionTest.java

```diff
@@ -140,13 +142,14 @@ public boolean equalsWithoutChildren(@Nonnull PlannerExpression otherExpression)
         }
     };
 
-    @Test
-    public void testOneOfThemEqualsNoValues() throws Exception {
+    @ParameterizedTest(name = "testOneOfThemEqualsNoValues [emptyIsUnknown = {0}]")
+    @BooleanSource
+    public void testOneOfThemEqualsNoValues(boolean emptyIsUnknown) throws Exception {
         final TestScalarFieldAccess noRepeatedValues = TestScalarFieldAccess.newBuilder().build();
-        final QueryComponent component = field("repeat_me").oneOfThem().equalsValue("fishes");
+        final QueryComponent component = field("repeat_me").oneOfThem(emptyIsUnknown).equalsValue("fishes");
```

Comment:
I guess we should also test that it still works as expected if there are matches, i.e., that if there is a field, that setting `emptyIsUnknown` to `false` still returns `Boolean.TRUE`. Not sure if there's already a test that could be parameterized.

Reply:
Added a new one.